platform/ui/src/components/Typography/Typography.jsx

```diff
@@ -0,0 +1,142 @@
+import React from 'react';
```

Comment:
Is the goal here to duplicate React Material's Typography Component?

- https://material-ui.com/components/typography/
- https://material-ui.com/api/typography/